src/runtime/miscellaneous.rs

```diff
@@ -76,6 +77,22 @@ pub struct Mount {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     /// Options are fstab style mount options.
     options: Option<Vec<String>>,
+
+    #[serde(
+        default,
+        skip_serializing_if = "Option::is_none",
+        rename = "uidMappings"
+    )]
+    /// UID mappings used for changing file owners w/o calling chown, fs should support it. Every mount point could have its own mapping.
```

Comment:
How about being a bit more verbose here:
```suggestion
   /// UID mappings for ID-mapped mounts (Linux 5.12+).
   ///
   /// Specifies how to map UIDs from the source filesystem to the destination mount point.
   /// This allows changing file ownership without calling chown.
   ///
   /// **Important**: If specified, gid_mappings MUST also be specified.
   /// The mount options SHOULD include "idmap" or "ridmap".
   ///
   /// See: https://github.com/opencontainers/runtime-spec/blob/main/config.md#posix-platform-mounts
```